dev/io.openliberty.persistence.3.1.thirdparty/build.gradle

```diff
@@ -55,9 +55,7 @@ task validateNLS {
         fileset(dir: existingResourceDir) {
           include(name: '**/*Resource.nlsprops')
           exclude(name: '**/TraceLocalizationResource.nlsprops')
-          exclude(name: '**/*Resource_*.nlsprops')
```

Comment:
Please keep these excludes, but change the comments to indicate they need to be removed with 4.0.0.5 update